internal/pkg/cli/env_package.go

```diff
@@ -92,7 +93,7 @@ func newPackageEnvOpts(vars packageEnvVars) (*packageEnvOpts, error) {
 		ws:           ws,
 		sel:          selector.NewLocalEnvironmentSelector(prompt.New(), cfgStore, ws),
 		caller:       identity.New(defaultSess),
-		fs:           &afero.Afero{Fs: afero.NewOsFs()},
+		fs:           fs,
```

Comment:
This is a refactor unrelated to the main purpose of this PR.
 
The `fs` field of `packageEnvOpts` needs to just implement `afero.Fs`, and `afero.NewFs()` already does this. We don't need the other methods in `*afero.Afero`.